### PR TITLE
[TEST ME] add most of a new !bug command

### PR DESCRIPTION
**Pull request by Velfi**
add new help command

### CODE REVIEW

**Comment by randomPoison** [comment]
I'm pretty sure I like this, but you're gonna have to rewrite this to use [Poise](https://github.com/serenity-rs/poise) since I'm getting the project setup to use it. I'll update `main` to use it shortly, then you can rebase/merge and rewrite as needed.

**Comment by randomPoison** [comment]
Okay `main` has been updated to use Poise, so you'll want to rebase and rework the `.bug` code to follow the structure that `.todo` is using.

File: src/bug.rs
Comment (randomPoison) [inline src/bug.rs:40]: You can derive `Default` for enums now using the `#[default]` attribute: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=e460b76ed5641af636e956fb61d0fc3d

File: src/bug.rs
Comment (randomPoison) [inline src/bug.rs:68]: I don't think we need all three of these. I was thinking of modeling our bugs along the lines of the GH issue tracker, so just a title and a description.

File: src/bug.rs
Comment (randomPoison) [inline src/bug.rs:65]: Can we actually hold off on adding labels? I also want labels, but some thought needs to be put into the design of them (e.g. I kinda of want them to be UUID backed so that we can easily rename/move/manage labels), so lets start without them and worry about them if it's really annoying to not have them. I hit a similar concern adding categories for `.todo`, but in that case each item only has one category and here we have to assume that bugs will have multiple labels.

File: src/bug.rs
Comment (randomPoison) [inline src/bug.rs:70]: I think let's also skip `plus_ones` for now. If we go in that direciton we probs wanna support a more generic emoji reaction system rather than hard coding the concept of "plus ones".

**Review by randomPoison — APPROVED** [review]
I guess I left some comments but tbh I kinda wanna just merge this as-is and iterate on it more myself. But is your PR so it up to you: Do you wanna iterate on this more in a PR or do you want to merge as-is and let me finish polishing this up?

**Comment by Velfi** [comment]
> I guess I left some comments but tbh I kinda wanna just merge this as-is and iterate on it more myself. But is your PR so it up to you: Do you wanna iterate on this more in a PR or do you want to merge as-is and let me finish polishing this up?

You're the boss + you have more time for this than me so be my guest. LMK if you have trouble updating the Pest grammar.

**Comment by randomPoison** [comment]
I'm going to merge this into a branch on this repo and then iterate on it a bit before merging. Thanks for getting a first draft together!

**Pull request merged**
